docs/01-app/03-api-reference/02-components/image.mdx

```diff
@@ -84,6 +84,10 @@ export default function Page() {
 }
 ```
 
+> **Good to know**: When specifying an external URL or API Routes (internal URL) as the image source, the image may be processed through Next.js built-in [Image Optimization](/docs/app/getting-started/images).
```

Comment:
```suggestion
> Good to know: When Next.js Image Optimization is used, for security reasons, request headers are not forwarded to the image source. If retrieving the image requires authentication, consider using the [unoptimized](#unoptimized) property to disable Image Optimization.
```

or if we still want to mention the "internal URL" caveat:

```suggestion
> Good to know: When Next.js Image Optimization is used, request headers are not forwarded to the image source, even when that source is your own Next.js server (API Routes or assets from public folder). If retrieving the image requires authentication, consider using the [unoptimized](#unoptimized) property to disable Image Optimization.
```

Also the image optimization link was not pointing at relevant docs